Document agent environment variables (#1237)
sensu/sensu-docs: content/sensu-go/5.2/reference/agent.md

See the example config file provided with Sensu at `/usr/share/doc/sensu-go-agent-5.2.1/agent.yml.example`.
The agent loads configuration upon startup, so you must restart the agent for any configuration updates to take effect.

Sensu agents can also be configured using environment variables.
With the exception of `labels`, all agent configuration flags can be applied using environment variables in the format `SENSU_$FLAG_NAME` where `$FLAG_NAME` is the name of the configuration flag.
Environment variables syntax depends on type; configuration flags of type array must use a space-separated list.
For example:

{{< highlight shell >}}
# string | integer | boolean

> export SENSU_NAME=webserver01

Author: remove exports

Reviewer: Maybe add note that this pertains primarily to a container workflow?

See the example config file provided with Sensu at `/usr/share/doc/sensu-go-agent-5.2.1/agent.yml.example`.
The agent loads configuration upon startup, so you must restart the agent for any configuration updates to take effect.

Sensu agents can also be configured using environment variables.

> Environment variables can be a useful configuration tool for container-based workflows.

Author: ```suggestion
Environment variables can be a useful configuration tool when deploying agents in containers.
```
